velox/docs/functions/spark/datetime.rst

```diff
@@ -76,6 +76,12 @@ These functions support TIMESTAMP and DATE input types.
 
     This is an alias for :spark:func:`dayofweek`.
 
+.. spark:function:: date_from_unix_date(integer) -> date
```

Comment:
Please make it in alphabetical order according to function name.

Reply:
Updated, thanks!